pinecone/core/client/model_utils.py

```diff
@@ -63,7 +61,7 @@ def __get__(self, instance, cls=None):
             return result
 
 
-PRIMITIVE_TYPES = (list, float, int, bool, datetime, date, str, file_type)
+PRIMITIVE_TYPES = (list, float, int, bool, str, file_type)
```

Comment:
Should we disable file type? Seems like a potential unnecessary conversion.

Reply:
Yeah, I can yank that out too easily enough.

Reply:
Actually, even though I think this should be safe to do, references to `file_type` are spread across a couple different files. So I will make a different ticket to handle that a bit later.